gulpfile.js

```diff
@@ -41,7 +41,8 @@ gulp.task('cards', () => (
     for (const file of files) { // eslint-disable-line
       const f = fs.readFileSync(`./cards/${file}`, { encoding: 'utf8' });
       const parsed = frontmatter(f);
-      cards.push(buildCardObject(file, parsed.attributes, parsed.body));
+      const body = parsed.body.replace(/^#(?!#).+/m, ''); // remove titles from body
```

Comment:
This strips out the first `<h1>` from the markdown so that we don't have to `display: none` it after rendering it